metnet/models/metnet2.py

```diff
@@ -38,32 +39,33 @@ def __init__(
         **kwargs,
     ):
         """
-        MetNet-2 builds on MetNet-1 to use an even larger context area to predict up to 12 hours ahead.
+        MetNet-2 extends MetNet-1 to use an even larger context area to predict from 12 hours ahead.
 
         Paper: https://arxiv.org/pdf/2111.07470.pdf
 
-        The architecture of MetNet-2 differs from the original MetNet in terms of the axial attention is dropped, and there
-        is more dilated convolutions instead.
+        The architecture of MetNet-2 differs from the original MetNet in terms of the axial
+        attention is dropped, and there is more dilated convolutions instead.
 
         Args:
-            image_encoder:
-            input_channels:
-            input_size:
-            lstm_channels:
-            encoder_channels:
-            upsampler_channels:
-            lead_time_features:
-            upsample_method:
-            num_upsampler_blocks:
-            num_context_blocks:
-            num_input_timesteps:
-            encoder_dilations:
-            sat_channels:
-            output_channels:
-            kernel_size:
-            center_crop_size:
-            forecast_steps:
-            **kwargs:
+            image_encoder: string
```

Comment:
Ideally, add the description for what these args are, like in the above ones. But there are a lot here, so feel free to leave this for this one.